docs/sphinx/user_guide/feature/reduction.rst

```diff
@@ -159,3 +159,108 @@ Reduction Policies
 For more information about available RAJA reduction policies and guidance
 on which to use with RAJA execution policies, please see 
 :ref:`reducepolicy-label`.
+
+--------------------------------
+Experimental Reduction Interface
+--------------------------------
+
+An experimental reduction interface is now available that hopes to improve
```

Comment:
Can we add a paragraph to briefly explain how the new reducers work from the user perspective and how maybe how that differs from the old reducers?

Reply:
Done.

Reply:
@mdavis36 When describe the differences between the new interface and the old one, I think it would be good to add that no reduction policy is required for the new interface.

Reply:
Done